.ruby-version

```diff
@@ -1 +0,0 @@
-2.6
```

Comment:
wait why did we dump this?

Reply:
Looks like it went away in #4937

Reply:
It's coming back in https://github.com/18F/identity-idp/pull/4949

Reply:
I'll tear down the branch and create new from scratch once #4949 is in.

Reply:
Thanks! It's merged